script.js

```diff
@@ -1,11 +1,250 @@
+        technologies: ["HTML", "CSS", "JavaScript", "Ruby", "Bootstap"],
+        liveLink: "#",
+        sourceLink: "#"
+    },
+]
+
+
+const btnsDesk = document.querySelectorAll(".card-btn-disk")
+btnsDesk.forEach((btn) => {
+    btn.addEventListener("click", function (e) {
+        e.preventDefault();
+        const projectId = btn.id; // Get the ID of the clicked button
+        popUpDisk(projectId);
+    });
+});
+
+function popUpDisk(id){
+    console.log(id)
+    let selectedProject = diskProjects.find(project => project.id === id);
+    
+    if (!selectedProject) {
+        console.error("Project not found!");
+        return;
+    }
+    let projectPopUp = document.createElement("div");
+    projectPopUp.classList.add("disk-project-popup");
+    document.body.appendChild(projectPopUp)
+    projectPopUp.innerHTML = `
+        <div class="popup">
+            <div class="popup-content">
+                <a onclick="closePopup()" href="#"><i class="fa-solid fa-xmark" id="x-mark-disk"></i></a>
+                <h2 class="poppins-bold first-h1">${selectedProject.name}</h2>
+                <div class="card-meta">
+                   <div class="container">
+                      <span class="poppins-600 company-name">${selectedProject.company}</span>
+                      <span class="dot"></span>
+                      <span class="poppins-600 role">${selectedProject.role}</span>
+                      <span class="dot"></span>
+                      <span class="poppins-600 year">${selectedProject.year}</span>
+                   </div>
+                 </div>
+                <img src="${selectedProject.image}" alt="${selectedProject.name}" class="popup-image-disk">
+                <div class= "container-body">
+                    <div class= "description-disk">
+                         <p class="poppins-regular paraa">${selectedProject.description}</p>
+
+                    </div>
+                    <div class= "description-disk-2">
+                         <div class="poppins-mid card-tags">${selectedProject.technologies.map(tech => `<span class= "tag">${tech}</span>`).join('')}</div>
+                         <div class="hrr"></div>
+                         <div class="popup-button">
+                             <a href="${selectedProject.liveLink}" class="poppins-mid card-button" target="_blank" class="popup-button">See Live () </a>
+                             <a href="${selectedProject.sourceLink}" class="poppins-mid card-button" target="_blank" class="popup-button">Source Code <i class="fa-brands fa-github"></i></a>
+                         </div>
+                    </div>
+
+                </div>
+                  
+            </div>
+        </div>
+    `;
+    document.querySelector(".disk-project-popup").style.display = "block";
+    document.documentElement.classList.add("no-scroll");
+}
+function closePopup() {
```

Comment:
The name of this function should be different than closePopup. It can be closeDesktopPopup or something like that as these two functions are working differently.